src/content/samples/node.mdx

```diff
@@ -43,58 +61,7 @@ Log into [Okteto Cloud](https://cloud.okteto.com/#/?origin=docs) and click on th
 
 Did you notice that you're accessing your application through an HTTPs endpoint? This is because Okteto Cloud will [automatically create them](/docs/cloud/ssl/) for you when you deploy your application. Cool no 😎?
 
-## Step 2: Create your okteto manifest
-
-To start developing on the Node.js Sample App you first need to create an okteto manifest.
-With the Node.js Sample App deployed, run the following command to create your [okteto manifest](/docs/reference/manifest/):
-
-```console
-$ okteto init --v1
-```
-
-```console
-This command walks you through creating an okteto manifest.
-It only covers the most common items, and tries to guess sensible defaults.
-See https://okteto.com/docs/reference/manifest for the official documentation about the okteto manifest.
-Use the arrow keys to navigate: ↓ ↑ → ←
-Select the deployment you want to develop:
-  ▸ hello-world
-    Use default values
-```
-
-The `okteto init` command will scan the available deployments in your Kubernetes namespace and ask you to pick one.
-Select the `hello-world` deployment. It's the one we deployed on the previous step.
-
-```console
- ✓  hello-world
- ✓  Deployment 'hello-world' successfully analyzed
- ✓  okteto manifest (okteto.yml) created
- i  Run 'okteto up' to activate your development container
-```
-
-The `okteto init` command creates the following `okteto.yml` file:
-
-```yaml
-name: hello-world
-command: bash
-sync:
-  - .:/usr/src/app
-forward:
-  - 3000:3000
-  - 9229:9229
-```
-
-This file defines how to activate a development container for the Node.js Sample App:
-
-- `name`: the name of the Kubernetes deployment you want to put on development mode
-- `command`: the start command of the development container
-- `sync`: the folders that will be synchronized between your local machine and the development container
-- `forward`: a list of ports to forward from your development container
-
-Also, the `okteto init` command creates a `.stignore` file to indicate which files shouldn't be synchronized to your development container.
-This is useful to avoid synchronizing binaries, build artifacts, git metadata, or dependencies like `node_modules`.
-
-## Step 3: Activate your development container
+## Step 2: Activate your development container
```

Comment:
You are missing changes in this section:
https://github.com/okteto/docs/pull/119/files#diff-a7c7d0def22235aa498659f147beb3ee4800dfde172e7f81d3ab29244224de16R68